js/main.js

```diff
@@ -8,26 +8,51 @@ var $view = document.querySelectorAll('.view');
 var $entriesLink = document.querySelector('#entries-link');
 var $newLink = document.querySelector('#new-link');
 var $noEntry = document.querySelector('#no-entry');
+var $ul = document.querySelector('ul');
+var $titleInput = document.getElementById('title-input');
+var $notesInput = document.getElementById('notes-input');
+var $headerText = document.getElementById('header-text');
 
 $photoInput.addEventListener('input', handleInput);
 $createForm.addEventListener('submit', handleSubmit);
 window.addEventListener('DOMContentLoaded', handleLoad);
 $entriesLink.addEventListener('click', dataView);
 $newLink.addEventListener('click', dataView);
+$ul.addEventListener('click', editEntry);
 
-var $ul = document.querySelector('ul');
+function editEntry(event) {
+  var $dataView = event.target.getAttribute('data-view');
+  $headerText.textContent = 'Edit Entry';
+
+  if (event.target.nodeName === 'I' && $dataView !== '') {
+    viewSwap($dataView);
+  }
```

Comment:
This should be a guard before any of this happens, I think. If the user didn;t click an `i` element, none of the other stuff in this function should be happening either, right?